### DISCUSSION TITLE
v1.0.0atentdead0-94-g9b64d52c for m644P-LARSP

### DISCUSSION THREAD

**Discussion by NightriderC64** [thierer/sd2iec#4] [General]
Hi, 

I have SD2IEC identifying as SD2IEC V1.0.0ATENTDEAD0-29-M644-LARSP-MINI. It does not want to update with any of binaries in release v1.0.0atentdead0-94-g9b64d52c. Could you compile and share this release for M644-LARSP version? Is it possible at all?

Reards, 
Marcin

**Comment by thierer** [comment]
> Is it possible at all?

The upstream firmware already has a reduced feature set on the m644p because it otherwise won't fit the flash. Maybe if you're willing to give up on a lot of features you could compile your own, minimal firmware which still suits your needs, but I doubt it.

**Reply by NightriderC64** [reply]
OK, I see the problem. I think I'll better look for sd2iec with m1284. Thanks for reply.

**Reply by NightriderC64** [reply]
I've  got m1284 version of sd2iec. It has updated firmware without any problem. Works great! Nice to watch top demos on sd2iec :)
